packages/react-positioning/src/utils/mergeArrowOffset.ts

```diff
@@ -1,11 +1,14 @@
-import { Offset } from '@fluentui/react-positioning';
+import { Offset } from '../types';
 
 /**
+ * Generally when adding an arrow to popper, it's necessary to offset the position of the popper by the
+ * height of the arrow. A simple utility to merge a provided offset with an arrow height to return the final offset
+ *
  * @param userOffset - The offset provided by the user
  * @param arrowHeight - The height of the arrow in px
  * @returns User offset augmented with arrow height
  */
-export function getOffsetWithArrow(userOffset: Offset | undefined | null, arrowHeight: number): Offset {
+export function mergeArrowOffset(userOffset: Offset | undefined | null, arrowHeight: number): Offset {
```

Comment:
Share this so that components that use an arrow can merge and arrow offset with user offset easily